client/app/components/app.component.ts

```diff
@@ -1,6 +1,45 @@
 import { Component } from '@angular/core';
+
+export class Test {
+  id: number;
+  ip: string;
+  port: string;
+}
+
+
 @Component({
   selector: 'my-app',
-  template: '<h1>My First Angular 2 App</h1>'
+  template: `<h1>UnLoadX </h1>
+  			<h4>Enter Server IP Address & Port Number:</h4>
+  			<input [(ngModel)]="ipAddress" placeholder="IP Address">
+  			<input [(ngModel)]="port" placeholder="Port">
+  			<button (click)= "onClick()"> Add IP & Port </button>
```

Comment:
remove onClick for each ip/port pair - instead just one function (onSubmit or something) that files all IP/ports to the server once the form is submitted